float64 xPos
float64 yPos
float64 thetaPos
float64 micHeight
---
string response